src/robotide/__init__.py

```diff
@@ -67,8 +70,16 @@
 def main(*args):
     noupdatecheck, debug_console, inpath = _parse_args(args)
     if len(args) > 3 or '--help' in args:
-        print __doc__
+        print(__doc__)
         sys.exit()
+    if '--version' in args:
+        try:
+            import version
```

Comment:
This should be `import .version` or `from robotide import version`. Relative imports are always confusing and, in case it matters, they aren't Python 3 compatible.